build.sbt

```diff
@@ -7,6 +7,8 @@ val reactJS              = "16.13.1"
 val FUILess              = "2.8.7"
 val kindProjectorVersion = "0.13.0"
 
+ThisBuild / Test / bspEnabled := false
```

Comment:
What prompted this?

Reply:
I'm trying to get metals to use bsp as it seems more reliable than bloop but it chokes on tests that use node modules